src/main/java/com/emc/mongoose/storage/driver/pravega/PravegaStorageDriver.java

```diff
@@ -456,7 +456,46 @@ void submitStreamOperation(final PathOperation streamOp, final OpType opType) {
 	}
 
 	void submitStreamCreateOperation(final PathOperation streamOp, final String nodeAddr) {
-		// TODO: Vlad, issue SDP-47
+		try {
+			val endpointUri = endpointCache.computeIfAbsent(nodeAddr, this::createEndpointUri);
+			val controller = controllerCache.computeIfAbsent(endpointUri, this::createController);
+			val scopeName = DEFAULT_SCOPE;
+			var streamName = streamOp.item().name();
+			if(streamName.startsWith(SLASH)) {
+				streamName = streamName.substring(1);
+			}
+			if(streamName.endsWith(SLASH) && streamName.length() > 1) {
+				streamName = streamName.substring(0, streamName.length() - 1);
+			}
+			val streamConfig = StreamConfiguration
+					.builder()
+					.scalingPolicy(scalingPolicy)
+					.streamName(streamName)
+					.scope(scopeName)
+					.build();
+			if(controller.createStream(streamConfig).get(controlApiTimeoutMillis, TimeUnit.MILLISECONDS)) {
```

Comment:
we could execute the operation asynchronously if we supply the callback instead
created a ticket for this:
https://mongoose-issues.atlassian.net/browse/PRAVEGA-62

Reply:
Ok, then it would be a next PR.